File: app/src/main/java/com/tydeya/familycircle/presentation/ui/managerpart/menu/abstraction/ManagerMenuView.java
package com.tydeya.familycircle.presentation.ui.managerpart.menu.abstraction;

import com.tydeya.familycircle.presentation.ui.managerpart.menu.details.recyclerview.ManagerMenuItemType;

public interface ManagerMenuView {

    void signOut();

    void openPage(ManagerMenuItemType managerMenuItemType);
}

File: app/src/main/java/com/tydeya/familycircle/presentation/ui/managerpart/menu/ManagerMenuView.kt
package com.tydeya.familycircle.presentation.ui.managerpart.menu

import com.tydeya.familycircle.presentation.ui.managerpart.menu.recyclerview.ManagerMenuItemType

interface ManagerMenuView {

    fun signOut()

    fun openPage(itemType: ManagerMenuItemType)

    fun showDialog(itemType: ManagerMenuItemType)
}
